src/main/java/com/solvd/tickets/service/impl/TicketServiceImpl.java

```diff
@@ -34,18 +34,20 @@ public Mono<Ticket> create(Ticket ticket) {
                 .bodyToMono(Integer.class);
         return placesMono.flatMap(value -> {
             if(value < ticket.getAmount()) {
-                return Mono.error(new NoFreePlacesException("There are only " + value + " free places"));
+                if(value == 0){
+                    kfProducer.send(ticket.getFilmId().toString());
+                }
+                return Mono.error(new NoFreePlacesException("There are not enough  free places"));
```

Comment:
`Places` as a noun, in this case, will be grammatically incorrect.
Correct way: place

Reply:
may be rename to AllTicketsSoldException?

Reply:
go ahead